app/feature/feature-movingflow/src/main/kotlin/com/hedvig/android/feature/movingflow/ui/summary/SummaryDestination.kt

```diff
@@ -193,6 +197,9 @@ private fun SummaryScreen(
       Spacer(Modifier.height(16.dp))
       Column(verticalArrangement = Arrangement.spacedBy(8.dp)) {
         QuoteCard(content.summaryInfo.moveHomeQuote)
+        for (addonQuote in content.summaryInfo.moveHomeQuote.relatedAddonQuotes) {
+          AddonQuoteCard(addonQuote)
+        }
         for (mtaQuote in content.summaryInfo.moveMtaQuotes) {
```

Comment:
could there also be addons for mta quotes in the future?

Reply:
Good question, let's ask I'd say. If the answer is yes, adding them here shouldn't be a big pr, probably just resting the same card here in the UI.

Reply:
Okay, leaving this as-is for now according to the discussion we had in standup. If this comes up in the backend API, we'll do it then